### ISSUE TITLE
Line wrapping in TimeVector, UDQ

### ISSUE THREAD

**Issue by berland** [OPM/opm-common#1948]
Long UDQ statements cause line-breaks when printed, 

```
import datetime
from opm.tools import TimeVector

with open("udq.sch", "w") as fh:
    fh.write("""
UDQ
  DEFINE FU_PWRI1 WWIR A_01 + WWIR A_02 + WWIR A_03 + WWIR A_04 + WWIR A_05 + WWIR A_06 + WWIR A_07 + WWIR A_08     + WWIR A_09 + WWIR A_10  + WWIR A_11 + WWIR A_12 + WWIR A_13 + WWIR A_14 + WWIR A_15 /
/
""")
schedule = TimeVector(datetime.datetime.now())
schedule.load("udq.sch", datetime.datetime.now())
print(schedule)
```
This code yields:
```
DATES
  22 'SEP' 2020/
/

UDQ
   DEFINE 'FU_PWRI1' WWIR A_01 + WWIR A_02 + WWIR A_03 + WWIR A_04 + WWIR A_05
   + WWIR A_06 + WWIR A_07 + WWIR A_08 + WWIR A_09 + WWIR A_10 +
   WWIR A_11 + WWIR A_12 + WWIR A_13 + WWIR A_14 + WWIR A_15 /
/
```
where the linebreaks in the processed `DEFINE` line gives errors similar to this from E100:
```
3066 READING UDQ    
@--  ERROR FROM PROCESSOR 1 AT TIME    11149.0   DAYS    (10-APR-2026):
@           TERMINATING SLASH MISSING FOR RECORD 1 OF UDQ DATA OR RECORD   
@           IS LONGER THAN 128 CHARACTERS. RECORD WILL BE IGNORED.         
```

Tested with pypi-opm (~opm-common master summer 2020)

**Comment by joakim-hove** [comment]
Hmm;

is the problem that the line is broken with a newline - or that the total number of characters up to the terminating `/` is too high - I would actually guess it is the latter; and then this an Eclipse problem. You could *try*
```
UDQ
    DEFINE 'FU_PWR1' SUM( WWIR A_* ) /
/
```

**Comment by berland** [comment]
Ok - my example extended 128 characters,  so that should trigger that error message anyway. But the origin of the error stems from a case where the DEFINE line has 113 characters, so there the linebreaks do trigger the error.

**Comment by joakim-hove** [comment]
> But the origin of the error stems from a case where the DEFINE line has 113 characters, so there the linebreaks do trigger the error.

OK - will look into it.

**Comment by joakim-hove** [comment]
OK - this should be fixed in #1950
